### ISSUE TITLE
Fix failing grunt-saucelabs CI tests

### ISSUE THREAD

**Issue by jbeard4** [axemclion/grunt-saucelabs#212]
Continuous integration seems to be failing for grunt-saucelabs:

<img src="https://www.dropbox.com/s/tcqd2jmb4za2hnq/Screenshot%202016-07-06%2007.36.48.png?dl=1">

**Comment by Jonahss** [comment]
Yeah, I think there's a single failing test. I'll take a look at it.

**Comment by Jonahss** [comment]
![screen shot 2016-07-06 at 3 16 49 pm](https://cloud.githubusercontent.com/assets/1521841/16636406/d274af2e-438c-11e6-8f08-71c276ede1eb.png)


There we go.
Basically I removed support for YUI Test, so I removed the troublesome test (there was a race condition with the frameworks reporting code).

Updated dependencies.

Dev dependencies still out of date. Waiting on updates to grunt-sauce-tunnel.

The sauce badge is still red. Due to a bug in saucelabs' api I think? Can look into it a bit more.
